trakt_scrobbler/notifier.py

```diff
@@ -14,6 +14,14 @@
         toaster = ToastNotifier()
     else:
         import subprocess as sp
+        try:
+            import gi
+            gi.require_version('Notify', '0.7')
+            from gi.repository import Notify
+            Notify.init(APP_NAME)
+            notifier = Notify.Notification.new("trakt-scrobbler")
```

Comment:
```suggestion
            notifier = Notify.Notification.new(APP_NAME)
```
Shouldn't this be `APP_NAME` too?

Reply:
Just some minor changes and we'll be good to go :)